mock/role/routes.ts

```diff
@@ -196,6 +196,12 @@ export const asyncRoutes = [
         name: 'BackToTopDemo',
         meta: { title: 'backToTop' }
       },
+      {
```

Comment:
See line 182-185, remove this and change line 183 to '/count-to'

Reply:
already edited